__tests__/app.test.js

```diff
@@ -123,3 +123,69 @@ describe('/api/articles', () => {
 			});
 	});
 });
+
+describe('/api/articles/:article_id/comments', () => {
+	test('GET 200: responds with an array of comments with the associated article id', () => {
+		return request(app)
+			.get('/api/articles/3/comments')
+			.expect(200)
+			.then(({ body }) => {
+				const { comments } = body;
+				expect(comments).toEqual([
```

Comment:
this way of testing the comments is fine but a little bit brittle. It would be better to check the length of them, then iterate over and check the article ids all match and that the comments have the right keys